src/Operation/Watch.php

```diff
@@ -317,7 +317,6 @@ final public function commandSucceeded(CommandSucceededEvent $event): void
      * Execute the operation.
      *
      * @see Executable::execute()
-     * @param Server $server
      * @return ChangeStream
```

Comment:
Consider adding a `ChangeStream` return type to `execute()` and removing this.

Reply:
See above - it would constitute a BC break.